docs/manual.md

```diff
@@ -505,8 +506,38 @@ State14 ==
 InvariantViolation == hasLicense /\ year - BIRTH_YEAR < LICENSE_AGE
 ```
 
+<a name="lookup"></a>
+## 6.3. Module lookup
+
+Apalache uses [the SANY
+parser](https://lamport.azurewebsites.net/tla/tools.html), which is the
+standard parser of TLC and TLA+ Toolbox. By default, SANY is looking for the
+modules in the current working directory and in the Java package
+`tla2sany.StandardModules`, which is usually provided by `tla2tools.jar` that is
+included in the Java classpath.
+
+Additionally, Apalache includes the directory `$APALACHE_HOME/src/tla` into the
+SANY lookup list via the Java system variable called `TLA-Library`. (The script
+`bin/apalache-mc` automatically sets `APALACHE_HOME` to its parent directory.)
+
+On top of that, Apalache adds all the directories specified in the environment
+variable `TLA_PATH` into the SANY lookup list. See [issue
+187](https://github.com/informalsystems/apalache/issues/187).
```

Comment:
This reads to me a bit more like implementor-focused documentation. We might get something a little more friendly for end users if we say something like:

```suggestion
In addition to the modules in the current working directory, Appalache provides

- a small standard library (located in `$APALACHE_HOME/src/tla`), and
- support for additional source directories specified in the environment variable `TLA_PATH`. `TLA_PATH` should be a list of paths to directories separated by `:`. 

(Directories in the `TLA_PATH` are provided to SANY via the `TLA-Library` Java system variable.)    
```